FrontDesk/src/FrontDesk.Api/OfficeSettings.cs

```diff
@@ -0,0 +1,11 @@
+﻿using FrontDesk.Core.Interfaces;
+using System;
+
+namespace FrontDesk.Api
+{
+    public class OfficeSettings : IApplicationSettings
+    {
+        public int ClinicId { get { return 1; } }
+        public DateTime TestDate { get { return new DateTime(2014, 6, 9); } }
```

Comment:
Update this to 23 Sep 2030 (should be a Monday).

Reply:
Done

Reply:
That use to load appointments on that date so will change the service to get the data for the day selected.